aiohttp/web_fileresponse.py

```diff
@@ -37,6 +38,14 @@
 
 NOSENDFILE: Final[bool] = bool(os.environ.get("AIOHTTP_NOSENDFILE"))
 
+# File extensions for IANA encodings that will be checked in the order defined.
+# TODO(py311): Replace Literal with ContentCoding as StrEnum and remove .value
+ENCODING_EXTENSION: Final[dict[Literal["br", "gzip"], str]] = {
```

Comment:
```suggestion
ENCODING_EXTENSION: Final[Dict[str, str]] = {
```

I'm not sure we need `Literal` here as it would mean we would have every possible value that gets added to this in the type.